corehq/apps/app_manager/README.md

```diff
@@ -58,3 +58,21 @@ Linked apps use similar workflows to app copy for creating and pulling. See [doc
 Shadow forms are a variation of advanced forms that "shadow" another form's XML but can have their own settings and
 actions. Because they don't have their own XML, shadow forms do not have an xmlns but instead inherit their
 source form's xmlns. In reports, submissions from shadow forms show up as coming from their source form.
+
+# Features
+
+## Template apps
+
+HQ currently has two versions of "template apps."
+
+### Onboarding apps
+
+The first set of template apps are simple apps in different seectors that we've experimented with adding to a user's project when they first sign up for an account. These template apps are stored as json in the code base, in the [template_apps](https://github.com/dimagi/commcare-hq/tree/master/corehq/apps/app_manager/static/app_manager/template_apps) directory. They are imported using the view `app_from_template`.
```

Comment:
(Not sure if it's worth the effort to fix :upside_down_face: but there is a typo in "seectors".)

Reply:
Sure, https://github.com/dimagi/commcare-hq/pull/27140